LayoutTests/imported/w3c/web-platform-tests/css/css-transforms/animation/rotate-animation-on-svg.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<title>Animating the "rotate" property on an SVG element</title>
+<link rel="help" href="https://drafts.csswg.org/css-transforms-2/#individual-transforms">
+<link rel="match" href="rotate-animation-on-svg-ref.html">
+
+<style>
+
+@keyframes rotate-animation {
+    from { rotate: 0; }
+    to   { rotate: 180deg; }
```

Comment:
How can you tell this animation happened? 180deg on a 100x100 rectangle is the same as not doing anything. Shouldn't you use 45deg?

Reply:
The `transform-origin` makes it so that it's distinguishable.